test/framework/build.gradle

```diff
@@ -39,6 +39,8 @@ dependencies {
   api "com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}"
   api "org.apache.commons:commons-compress:1.19"
   api "org.apache.commons:commons-lang3:${versions.commons_lang3}"
+
+  runtimeOnly project(path: ":libs:elasticsearch-x-content:impl", configuration: "providerDir")
```

Comment:
Why are we using the providerDir configuration here? When does the provider-jars.txt file come into play?

Reply:
Oh, I see now. This is due to how we find stuff via relative paths. So we expect there to be some folder that includes the x-content jar plus all the other stuff it needs. Ug, that's unfortunate.